packages/react-components/src/index.ts

```diff
@@ -55,7 +73,42 @@ export type {
   StrokeWidthTokens,
   Theme,
 } from '@fluentui/react-theme';
-export { SSRProvider } from '@fluentui/react-utilities';
+export {
+  clamp,
+  getNativeElementProps,
+  getNativeProps,
+  getPartitionedNativeProps,
+  getRTLSafeKey,
+  getSlots,
+  omit,
+  resolveShorthand,
+  shouldPreventDefaultOnKeyDown,
+  useBoolean,
+  useConst,
+  useControllableState,
```

Comment:
```suggestion
```

`useControllableState` is quite complex and really only useful from the library perspective because we want users to be able to use a component in controlled and uncontrolled mode. It would be extremely niche to have this in an app since they would normally be hooked up to some sort of state management